students/OwenGaustad/Lab2/AnalogSensors/analogSensor_basics/analogSensor_basics.ino

```diff
@@ -0,0 +1,118 @@
+const int apin = 0;
+const int ledPin1 = 13;
+const int ledPin2 = 11;
+const int ledPin3 = 9;
+const int ledPin4 = 7;
+const int ledPin5 = 5;
+const int ledPin6 = 4;
+const int ledPin7 = 3;
+const int ledPin8 = 2;
+int val = 0;
+int brightness = 0;
+float voltage = 5.07;
+int resistor = 10000;
+
+
+void setup() {
+  digitalWrite(ledPin1, OUTPUT);
+  digitalWrite(ledPin2, OUTPUT);
+  digitalWrite(ledPin3, OUTPUT);
+  digitalWrite(ledPin4, OUTPUT);
+  digitalWrite(ledPin5, OUTPUT);
+  digitalWrite(ledPin6, OUTPUT);
+  digitalWrite(ledPin7, OUTPUT);
+  digitalWrite(ledPin8, OUTPUT);
+  Serial.begin(9600);
+}
+
+void checkSensor()  {
+  double val = log(analogRead(apin));
+  Serial.println(val);
+  if (val<1.8){
+    digitalWrite(ledPin1, HIGH);
+    digitalWrite(ledPin2, LOW);
+    digitalWrite(ledPin3, LOW);
+    digitalWrite(ledPin4, LOW);
+    digitalWrite(ledPin5, LOW);
+    digitalWrite(ledPin6, LOW);
+    digitalWrite(ledPin7, LOW);
+    digitalWrite(ledPin8, LOW);
```

Comment:
With a little effort I bet you could find a way to avoid the large `if` blocks with (mostly) repeated code.